src/autoscaler/db/sqldb/appmetric_sqldb_test.go

```diff
@@ -76,8 +76,8 @@ var _ = Describe("AppMetricSQLDB", func() {
 			BeforeEach(func() {
 				appMetric := &models.AppMetric{
 					AppId:      "test-app-id",
-					MetricType: models.MetricNameMemory,
-					Unit:       models.UnitMegaBytes,
+					MetricType: "test-metric-name",
```

Comment:
The two strings "test-metric-name" and "test-metric-unit" appear many times in these test cases, is it better to define them as constant?